ios/FluentUI/Navigation/UIBarButtonItem+BadgeValue.swift

```diff
@@ -33,4 +34,16 @@ import UIKit
             badgeValue = nil
         }
     }
+
+    /// Set this format string in a way that takes two parameters: item's accessibility label and badge value.
+    /// Format: %@ %@ <some string>
+    /// Example: "<Activity> <5> new activities" which will be called by voice over as "Activity 5 new activities button"
+    @objc var accessibilityLabelFormatString: String? {
```

Comment:
I don't understand this exposure. item already has accessibilitylabel that clients can override. why do we need to expose yet another label format string? we should be careful any additional public function especially in extension files

Reply:
Actually, the client can set accessibility label as just the name of the button and that will be called when there is only a button present without badge value and if there is badge value on that button then this string can be used or the default one which I added in the Localization file.

If I will not expose this then the client can only set one string as an accessibility label string and that will be used with and without a badge. So voice over will speak -> "Dismiss 0 items button" and "Dismiss 3 items button"

Reply:
clients knows what the badge values are and they send that to fluent to update the badge. why can't they can't set it directly on the button with the badge value?

Reply:
So basically client can only set properties of UIBarButtonItem and internally we make UIButton from it and clients can set badge value on UIBarButtonItem at any point of time in the future which internally gets updated on UIButton by listening to "notification on a change in badge value"

How a client can directly inform this UIButton class about what kind of accessibility label format string should be set in two different scenarios without exposing this property.

Reply:
Understood!! Yes, it makes complete sense to ask the client the badge value and badge accessibility label together.